src/app/clusters/bridged-device-basic/bridged-device-basic.cpp

```diff
@@ -0,0 +1,25 @@
+/**
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+
+#include "bridged-device-basic.h"
+
+#include <app/util/af.h>
+
+using namespace chip;
+
+void emberAfPluginBridgedDeviceBasicServerInitCallback(void) {}
```

Comment:
If we don't need an init callback, why are we adding one?

Reply:
removed old placeholder